docker-compose.yml

```diff
@@ -90,6 +90,25 @@ services:
       - protectedplanet_bundler:/usr/local/bundle
     network_mode: host
 
+  api:
+    container_name: protectedplanet-api
+    build: ${API_ABSOLUTE_PATH}
+    command: /bin/bash -l -c "bundle exec rackup"
+    volumes:
+      - ${API_ABSOLUTE_PATH}:/ProtectedPlanetApi
+      - ${SSH_AUTH_SOCK}:${SSH_AUTH_SOCK}
+    network_mode: host
+    environment:
+      - SSH_AUTH_SOCK=${SSH_AUTH_SOCK}
+    env_file:
+      - ${API_ABSOLUTE_PATH}/.env
+    depends_on:
+      - db
+    stdin_open: true
+    tty: true
```

Comment:
not sure what these do

Reply:
needed for debugging, interactive console